### ISSUE TITLE
grpc gem never added to load path

### ISSUE THREAD

**Issue by thomasklinger1234** [rules-proto-grpc/rules_proto_grpc#65]
Hello, 
I am writing a Ruby app which should use GRPC. When I build the app using bazel, `grpc` gem is never added to the gems. In fact, when adding `grpc` as a dependency, I only get `googleapis-common-protos-types` but never `grpc`.  I am not sure if this related to `rules_proto_grpc` or `bazelruby_rules_ruby` to be honest.

My program:

```
#!/usr/bin/env ruby

puts "GEM_PATH: #{ENV["GEM_PATH"]}"
puts "GEM_HOME: #{ENV["GEM_HOME"]}"
Dir[ENV["GEM_HOME"] + "/**/*"].each do |p|
    puts "  #{p}"
end
```

This never includes the `grpc` gem. 

### Gemfile

```
# frozen_string_literal: true

source "https://rubygems.org"

git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

gem 'grpc', '1.25.0'
gem 'multi_json', '1.13.1'
gem "google-protobuf", "3.11.0"

gem 'rspec', '~> 3.7.0'
gem 'rspec-its'
gem 'rspec-mocks'
gem 'rubocop', '~> 0.78.0'
gem 'rake'
```

### WORKSPACE

```
[...]

ruby_bundle(
    name = "rules_proto_grpc_gems",
    gemfile = "@rules_proto_grpc//ruby:Gemfile",
    gemfile_lock = "@rules_proto_grpc//ruby:Gemfile.lock",
)

ruby_bundle(
    name = "bundle",
    gemfile = "//tools/dependencies:Gemfile",
    gemfile_lock = "//tools/dependencies:Gemfile.lock",
    visibility = ["//visibility:public"],
)
```

### BUILD.bazel

```
package(default_visibility = ["//visibility:public"])

load(
    "@bazelruby_rules_ruby//ruby:defs.bzl",
    "ruby_binary",
)

ruby_binary(
    name = "app",
    srcs = [
        "main.rb",
        "//services/myruby/protos:ruby_grpc_library",
    ],
    main = ":main.rb",
    deps = [
        "@rules_proto_grpc_gems//:grpc", <- This does not work
        "@bundle//:grpc", <- This only adds googleapis-common-protos-types
    ],
)
```

**Comment by aaliddell** [comment]
I believe this is fixed in #106, since up to now we have used the older ruby rules set

**Comment by aaliddell** [comment]
Not fixed in dev, blocked upstream https://github.com/bazelruby/rules_ruby/issues/86

**Comment by aaliddell** [comment]
This will be fixed in 4.0.0
